flask_admin/contrib/peewee/view.py

```diff
@@ -213,10 +213,8 @@ def init_search(self):
                 if isinstance(p, string_types):
                     p = getattr(self.model, p)
 
-                field_type = type(p)
-
                 # Check type
-                if (field_type != CharField and field_type != TextField):
+                if not isinstance(p, (CharField, TextField, )):
```

Comment:
Can you change this from `(CharField, TextField, )` to `(CharField, TextField)`?